docs/list_of_apis.jl

```diff
@@ -15,6 +15,8 @@ apis = [
         "Land Model" => "APIs/Land/LandModel.md",
         "Soil Water Parameterizations" =>
             "APIs/Land/SoilWaterParameterizations.md",
+        "Soil Heat Parameterizations" =>
+            "APIS/Land/SoilHeatParameterizations.md",
```

Comment:
https://github.com/CliMA/ClimateMachine.jl/pull/1442/checks?check_run_id=986811966#step:8:94

(lowercase s in APIs)

Reply:
done!